app/models/category.rb

```diff
@@ -0,0 +1,12 @@
+class Category
+  include Mongoid::Document
+  has_many :words
+
+
+  field :name_de, type: String
+  field :name_en, type: String
+  field :description_de, type: String
+  field :description_en, type: String
+  field :type_de, type: String
```

Comment:
Why do we need a type for each language? The type should be the same for all languages. Or am I missing something. BTW: you cannot use the word 'type' as it is a reserved word. Use something like word_type.